src/ESM.sol

```diff
@@ -127,12 +134,14 @@ contract ESM {
     }
 
     function denyProxy(address target) external {
+        require(live == 1, "ESM/permanently-disabled");
         require(Sum >= min,  "ESM/min-not-reached");
 
         DenyLike(target).deny(proxy);
     }
```

Comment:
Unrelated to this change but we probably want a `DenyProxy` event here.
Maybe it's a good chance to add it since we are changing the function name on this deploy.

Reply:
I agree with this. Strictly speaking it should probably be a separate PR though.

Reply:
https://github.com/makerdao/esm/pull/26